notebooks/STIS/drizpac_notebook/STIS_DrizzlePac_Tutorial.ipynb

```diff
@@ -0,0 +1,1546 @@
+{
```

Comment:
<p>When attempting to extract files for ref_cti from the temp folder, the call on stsci(...) results in a FileNotFoundError. </p><pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
/var/folders/m8/sgnbqc5104q_68bvvrnnzn7m0005k0/T/ipykernel_55696/656885012.py in &lt;module&gt;
      5     ref_dir=os.path.abspath(ref_cti),
      6     num_processes=15,
----&gt; 7     verbose=2)

~/opt/anaconda3/envs/stscti/lib/python3.7/site-packages/stis_cti/stis_cti.py in stis_cti(science_dir, dark_dir, ref_dir, num_processes, pctetab, all_weeks_flag, allow, clean, clean_all, crds_update, ignore_missing, verbose)
    235     # Check science files for uncorrected super-darks:
    236     populate_darkfiles(raw_files, dark_dir, ref_dir, pctetab, num_processes, all_weeks_flag, 
--&gt; 237         clean_all, crds_update, ignore_missing, verbose)
    238     log.flush()
    239 

~/opt/anaconda3/envs/stscti/lib/python3.7/site-packages/stis_cti/stis_cti.py in populate_darkfiles(raw_files, dark_dir, ref_dir, pctetab, num_processes, all_weeks_flag, clean_all, crds_update, ignore_missing, verbose)
   1116         else:
   1117             clean_this = False
-&gt; 1118         generate_basedark(files, basedark, pctetab, num_processes, clean_this, verbose)
   1119 
   1120         # Make weekdark:

......

FileNotFoundError: [Errno 2] No such file or directory: 'cs0.e': 'cs0.e'
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
This means the `hstcal` dependency isn't installed.  `stistools.calstis.calstis()` calls the compiled c code `cs0.e` through a subprocess call.

Reply:
I have the `hstcal` version 2.7.4 installed in the environment.

Reply:
I see the issue was with my environment. Starting from a new env solved the problem on my end.